tests/testthat/test-qenv_eval_code.R

```diff
@@ -72,11 +72,15 @@ testthat::test_that("eval_code works with quoted code block", {
     q1@code,
     "a <- 1\nb <- 2"
   )
-  testthat::expect_equal(q1@env, list2env(list(a = 1, b = 2)))
+  testthat::expect_equal(q1@.xData, list2env(list(a = 1, b = 2)))
 })
 
 testthat::test_that("eval_code fails with unquoted expression", {
-  testthat::expect_error(eval_code(qenv(), a <- b), "object 'b' not found")
+  b <- 3
+  testthat::expect_error(
+    eval_code(qenv(), a <- b),
+    "unable to find an inherited method for function .eval_code. for signature"
+  )
```

Comment:
I don't understand what have changed that you have added `b <- 3`. In general I think test is not needed at all. `eval_code` supports certain types of inputs, providing failing call `a <- b` doesn't make sense - we don't need to test R execution precedence

Reply:
Yes, let's delete 👍